tools/docker/fabric-all-in-one/run-fabric-network.sh

```diff
@@ -49,6 +49,12 @@ function main()
     /bootstrap.sh ${FABRIC_VERSION} ${CA_VERSION} -b -s
     # Major version is 1.x or earlier (assumption is 1.4.x only)
     cd /fabric-samples/fabcar/
+
+    # PATCH
+    # This will change endorsment policy on fabcar chaincode.
+    # cactus-plugin-ledger-connector-fabric-socketio supports only single peer endorsment at the moment.
+    sed -i "s/AND('Org1MSP.member','Org2MSP.member')/OR('Org1MSP.member','Org2MSP.member')/g" ./startFabric.sh
```

Comment:
@outSH Could you please put this behind an `CACTUS_TEST_LOOSE_MEMBERSHIP_CHECK=ON/OFF` environment variable that is off by default? 
My concern is that the existing tests using the image could get compromised because of the altered ledger behavior. With the env var you can have it enabled when you need to but otherwise the change is backward compatible by default.

Reply:
@petermetz I thought there are no tests using faio 1.4 (since it didn't work for few months), but that's a good suggestion either way, will do 👍

Reply:
@outSH The tests are pinned to certain know-to-work versions of the AIO images for this exact reason. With that said, my memories are foggy about which ones are being skipped at present due to flakiness, so on top of what I just said it is still possible that the 1.4 test has not been part of the CI in recent months (but even if this is the case they'll be brought back on as soon as we can get rid of the flakes)

Thank you very much for the update, cheers!